model/nahled.php

```diff
@@ -70,34 +77,52 @@ function pokryjOrez($s, $v)
         return $this->mod($s, $v, self::POKRYJ_OREZ);
     }
 
-    /** Uloží stávající soubor s požadovanými úpravami */
+    /** Uloží stávající soubor s požadovanými úpravami do WebP formátu */
     protected function uloz(string $cil)
     {
-        $o = Obrazek::zSouboru($this->soubor);
-        $s = $this->s ?: 10000;
-        $v = $this->v ?: 10000;
-        switch ($this->mod) {
-            case self::PASUJ:
-                $o->fitCrop($s, $v);
-                break;
-            case self::POKRYJ:
-                $o->fillCrop($s, $v);
-                break;
-            case self::POKRYJ_OREZ:
-                $o->fill($s, $v);
-                break;
+        try {
+            $imagick = new Imagick($this->soubor);
+
+            $s = $this->s ?: $imagick->getImageWidth();
+            $v = $this->v ?: $imagick->getImageHeight();
+            
+            $s = max(1, (int)$s);
+            $v = max(1, (int)$v);
+
+            if ($this->mod) {
+                switch ($this->mod) {
+                    case self::PASUJ:
```

Comment:
Původně bylo `PASUJ` = `POKRYJ`.

Teď je to `POKRYJ` = `POKRYJ_OREZ`.

Je to ok?